rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java

```diff
@@ -38,10 +38,13 @@
 
     @Override
     public Response logbookQuery(HttpServletRequest request, LogBookQueryParams params) {
-        // TODO discuss about if is ok or not allow only to the root user to see the logs.
```

Comment:
Other than root user is out of scope for now.

Reply:
cc @jcabrerizo

Reply:
Good to me @algairim. Thanks

Reply:
@jcabrerizo , I tagged you to keep these changes in your records. Please do not waste your personal time on this review :)